packages/eslint-config-custom/index.js

```diff
@@ -3,7 +3,6 @@ module.exports = {
         'plugin:@typescript-eslint/recommended',
         'next',
         'next/core-web-vitals',
-        'turbo',
```

Comment:
You might also want to remove the dependency: https://github.com/workfloworchestrator/orchestrator-ui-library/blob/main/packages/eslint-config-custom/package.json#L14

Reply:
Done